client/src/partials/services/services.js

```diff
@@ -21,6 +22,12 @@ function ServicesController(Services, $translate, SessionService, ModalService,
   vm.cancel = cancel;
   vm.submit = submit;
   vm.del    = del;
+  vm.more   = more;
+
+
+  function handler(error) {
+    vm.state.error();
```

Comment:
From what I can see here `vm.state` is a string. This was used more in modules before `ui-router` was introduced. 

What does it so here to call `error()` on a string? 

I'm not sure this code would handle the error correctly.